drafts/prometheus-alertmanager-rhocp-open-liberty.adoc

```diff
@@ -0,0 +1,207 @@
++
+```
+global:
+  resolve_timeout: 5m
+  slack_api_url: <your slack webhook api url>
+
+# The root route. This route is used as the default
+# if there are no matches in the child routes.
+route:
+  group_by: [ alertname ]
+  group_wait: 10s
+  group_interval: 10s
+  repeat_interval: 5m
+  receiver: 'default_receiver'
+
+receivers:
+- name: 'default_receiver'
+  slack_configs:
+  - channel: 'prometheus-alertmanager-test'
+    title: "{{ range .Alerts }}{{ .Annotations.summary }}\n{{ end }}"
+    text: "*Description*: {{ .CommonAnnotations.description }}\n*Severity*: {{ .CommonLabels.severity }}"
+```
++
+In this example, we have one route that sends the alert to `default_receiver`. The receiver sends the alert to a Slack channel called `prometheus-alertmanager-test`.
+The `CommonAnnotations` come from the `annotations` you specified in your Prometheus Rule. The text is written using the link:https://golang.org/pkg/text/template/[Go templating] system.
++
+. On the RHOCP web console, go to Operators > Installed Operators > Prometheus Operator > Alertmanager and create a new Alertmanager instance. You do not need to change the default YAML.
+. Create a secret with your `alertmanager.yaml` file with the name of your alertmanager, prefixed by `alertmanager`:
++
+```
+❯ oc create secret generic alertmanager-alertmanager-main --from-file=alertmanager.yaml
+secret/alertmanager-alertmanager-main created
+```
++
+In this example, the name of our secret is `alertmanager-alertmanager-main` because our Alertmanager name is `alertmanager-main`, so we prefix the secret name with `alertmanager`.
++
+. Check that the service has started successfully.
++
+```
+❯ oc get svc -n prometheus-operator
+NAME                    TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                      AGE
+alertmanager-operated   ClusterIP   None         <none>        9093/TCP,9094/TCP,9094/UDP   18m
+prometheus-operated     ClusterIP   None         <none>        9090/TCP                     64m
+```
++
+. Once your pods are up and running, expose the route:
++
+```
+❯ oc expose svc/alertmanager-operated -n prometheus-operator
+route.route.openshift.io/alertmanager-operated exposed
+❯ oc get route -n prometheus-operator
+NAME                    HOST/PORT                                                                  PATH   SERVICES                PORT   TERMINATION   WILDCARD
+alertmanager-operated   alertmanager-operated-prometheus-operator.apps.jenniferc.os.fyre.ibm.com          alertmanager-operated   web                  None
+prometheus-operated     prometheus-operated-prometheus-operator.apps.jenniferc.os.fyre.ibm.com            prometheus-operated     web                  None
+```
+
+You can now access the Alertmanager UI. Since we haven't configured Prometheus to send the Alertmanager any alerts yet, you won't see any alert groups at the moment. We'll do that next.
```

Comment:
Change
"Since we haven't configured Prometheus to send the Alertmanager any alerts yet, you won't see any alert groups at the moment."
To
"Since we haven't yet configured Prometheus to send the Alertmanager any alerts, you won't see any alert groups at the moment."